hilti/src/rt/types/stream.cc

```diff
@@ -69,53 +53,181 @@ void Chunk::trim(Offset o) {
     _offset = o;
 }
 
-SafeConstIterator View::find(Byte b, const SafeConstIterator& n) const {
-    for ( auto i = UnsafeConstIterator(n ? n : _begin); i != UnsafeConstIterator(end()); ++i ) {
+const Chunk* Chain::findChunk(Offset offset, const Chunk* hint_prev) const {
```

Comment:
Can you add a test for this?

Reply:
Another a Chunk API piece, and it's implicitly tested through a bunch of other methods.